src/windows/FileTransferProxy.js

```diff
@@ -359,7 +359,13 @@ exec(win, fail, 'FileTransfer', 'upload',
                         if (params.hasOwnProperty(key)) {
                             var contentPart = new Windows.Networking.BackgroundTransfer.BackgroundTransferContentPart();
                             contentPart.setHeader("Content-Disposition", "form-data; name=\"" + key + "\"");
-                            contentPart.setText(params[key]);
+                            // CreateUploadSync fails when supplied with empty value for setText function. Pass " " instead when not specified.
+                            if (params[key]) {
+                                contentPart.setText(params[key]);
+                            }
+                            else {
+                                contentPart.setText(" ");
```

Comment:
That's an option. But we would lose sending the 'key' as well in that case. I thought this was the lesser of the two evils.